src/BassetServiceProvider.php

```diff
@@ -161,6 +164,23 @@ public function terminate(): void
         $basset->cacheMap->save();
     }
 
+    /**
+     * Loads needed basset disks.
+     *
+     * @return void
+     */
+    public function loadDisks(): void
+    {
+        // add the basset disk to filesystem configuration
+        app()->config['filesystems.disks.basset'] = [
+            'driver' => 'local',
+            'root' => storage_path('app/public'),
+            'url' => '/storage',
+            'visibility' => 'public',
+            'throw' => false,
+        ];
```

Comment:
What happens if someone already has a disk named `basset`?

Reply:
Let's only add it if it doesn't exist.

Reply:
done 🙌